charts/cluster-security/templates/roles.yaml

```diff
@@ -1,13 +1,27 @@
-apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: psp-baseline
-  labels:
-    kubernetes.io/cluster-service: "true"
-    eks.amazonaws.com/component: pod-security-policy
+  name: psp-baseline-clusterrole
```

Comment:
nit: the `-clusterrole` suffix is redundant since these names are already namespaced by kind. (similarly for the other one below)